doc/PostgreSQL.txt

```diff
@@ -0,0 +1,129 @@
+PostgreSQL Backend
```

Comment:
Possibly rename this file to `PostgreSQL_backend.txt` to make it clearer what it is about.